be/CMakeLists.txt

```diff
@@ -174,6 +174,17 @@ if (${JEMALLOC_DEBUG})
 endif()
 
 # Set all libraries
+add_library(ic STATIC IMPORTED)
```

Comment:
It's better to describe the solution and effect in the commit message.
1. The solution of inverted index.
2. Under inverted index, how much performance gained, how much storage cost.